src/fs-plus.coffee

```diff
@@ -4,6 +4,7 @@ path = require 'path'
 
 _ = require 'underscore-plus'
 async = require 'async'
+{Promise} = require 'es6-promise'
```

Comment:
For things like this that might be built-in depending on the context, we try to use the global when available and fallback to the require when it isn't there.

So something like:

```coffee
Promise = global.Promise ? require('es6-promise').Promise
```